tools/ci_connector_ops/ci_connector_ops/pipelines/actions/checks.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+"""This module groups functions made to run checks (static code analysis) for a specific connector given a test context."""
+
+from typing import List
+
+import asyncer
+from ci_connector_ops.pipelines.actions import environments
+from ci_connector_ops.pipelines.contexts import ConnectorTestContext
+from ci_connector_ops.pipelines.models import Step, StepResult
+from ci_connector_ops.pipelines.utils import get_step_result
+
+RUN_BLACK_CMD = ["python", "-m", "black", f"--config=/{environments.PYPROJECT_TOML_FILE_PATH}", "--check", "."]
```

Comment:
Do we have any worries about the `black` that we use in this dagger context ever becoming misaligned with the one that gradle runs? Inspired by [this question](https://airbytehq-team.slack.com/archives/C03VDJ4FMJB/p1678743659002559)

Reply:
It's a legit concern. I think we shall rely on a single version of the truth for black (and other tooling versions) using the root `pyproject.toml` file. But airbyte-python.gradle explicitely declares a black version. I copied it [here](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py#L15) for the "consistency" with what existed in the gradle pipeline. But it's indeed not the best approach in the long run. I'll create an issue.

Reply:
Thanks! I agree, `pyproject.toml` sounds like a good place for the truth